pan-domain-auth-verification/src/main/scala/com/gu/pandomainauth/service/CookieUtils.scala

```diff
@@ -28,7 +28,7 @@ object CookieUtils {
     AuthenticatedUser(
       user = User(data("firstName"), data("lastName"), data("email"), data.get("avatarUrl")),
       authenticatingSystem = data("system"),
-      authenticatedIn = Set(data("authedIn").split(",") :_*),
+      authenticatedIn = Set(data("authedIn").split(",").toSeq :_*),
```

Comment:
See https://docs.scala-lang.org/overviews/core/collections-migration-213.html#scalaseq-varargs-and-scalaindexedseq-migration.